src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -1018,39 +919,12 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
             const auto mlmodel_data_size =
                 static_cast<size_t>(mlmodel_data_end - mlmodel_data_begin);
 
-            void* alloc = nullptr;
-            const void* data = nullptr;
-            cuda_unique_ptr_ result;
-            switch (memory_type_) {
-                case TRITONSERVER_MEMORY_GPU: {
-                    call_cuda(cudaSetDevice)(memory_type_id_);
-                    call_cuda(cudaMalloc)(&alloc, mlmodel_data_size);
-                    call_cuda(cudaMemcpy)(
-                        alloc, mlmodel_data_begin, mlmodel_data_size, cudaMemcpyHostToDevice);
-                    result.reset(alloc);
-                    data = alloc;
-                    break;
-                }
-                case TRITONSERVER_MEMORY_CPU_PINNED: {
-                    call_cuda(cudaSetDevice)(memory_type_id_);
-                    call_cuda(cudaHostAlloc)(&alloc, mlmodel_data_size, cudaHostAllocPortable);
-                    call_cuda(cudaMemcpy)(
-                        alloc, mlmodel_data_begin, mlmodel_data_size, cudaMemcpyHostToHost);
-                    result.reset(alloc);
-                    data = alloc;
-                    break;
-                }
-                case TRITONSERVER_MEMORY_CPU:
-                default: {
-                    data = mlmodel_data_begin;
-                    break;
-                }
-            }
-
-            cxxapi::call(cxxapi::the_shim.InferenceRequestAppendInputData)(
-                request_, name_->c_str(), data, mlmodel_data_size, memory_type_, memory_type_id_);
-
-            return result;
+            cxxapi::call(cxxapi::the_shim.InferenceRequestAppendInputData)(request_,
+                                                                           name_->c_str(),
+                                                                           mlmodel_data_begin,
+                                                                           mlmodel_data_size,
+                                                                           TRITONSERVER_MEMORY_CPU,
```

Comment:
If the memory type is always CPU, wouldn't we never actually use the GPU? I had expected this to say GPU.